data/mods/Aftershock/items/afs_item_groups.json

```diff
@@ -55,27 +55,89 @@
     ]
   },
   {
-    "id": "schematics",
```

Comment:
Are you intending to remove the "schematics" itemgroup?  Because this seems like a merge error.

Reply:
That's a good point, I'll have a look as I don't think it's intentional.

Reply:
Just realised that my fork/branch of the master repo is now out of date after the various unused bots were depreciated from mainline into AFS. I'll re-add to reflect.

Reply:
Thanks for fixing that!